src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  const styleArray = sourceString.split(';');
+
+  styleArray.forEach((style) => {
+    const [property, value] = style.split(':').map((part) => part.trim());
+
+    if (property && value) {
+      styleObject[property] = value;
+    }
+  });
```

Comment:
The 'forEach' loop is used appropriately here to iterate over 'styleArray' and construct 'styleObject'. This adheres to the checklist item of avoiding loops and using iteration methods instead.